docs_src/src/components/documentation/Guides.jsx

```diff
@@ -34,6 +34,12 @@ const guides = [
     description:
       'Learn how to deploy your app to production and manage your deployments.',
   },
+  {
+    href: '/documentation/example_app/openapi',
```

Comment:
@VishnuSanal , this is a core feature. We should add it in the `api_reference` as well 😄